### PR TITLE
Update to stick to the spec and add SassDoc

### PR DESCRIPTION
**Pull request by kaelig**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
+.DS_Store
 .idea/
 .sass-cache/
 bower_components/
 build/
 demos/local/
-node_modules/
\ No newline at end of file
+node_modules/
+docs/sass
```
Comment (AlbertoElias) [inline .gitignore]: Make it docs/?

File: karma.conf.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ module.exports = function(config) {
 		// preprocess matching files before serving them to the browser
 		// available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
 		preprocessors: {
-			'test/*.test.js': ['browserify']
+			'test/**/*.js': ['browserify']
```
Comment (AlbertoElias) [inline karma.conf.js]: **/*.test.js to follow the convention

File: .gitignore (hunk 1 above)
Comment (kaelig) [inline .gitignore]: done!

File: karma.conf.js (hunk 2 above)
Comment (kaelig) [inline karma.conf.js]: Cheers, already fixed in 4a673a8

**Comment by AlbertoElias** [comment]
lgtm

**Pull request merged**